### ISSUE TITLE
icinga2 not acknowledging scheduled downtimes/ disabled notifications after HA is enabled for master zone

### ISSUE THREAD

**Issue by gangamaha** [Icinga/icinga2#7633]
## Describe the bug

We have 2 master nodes in master zone and bunch of satellite zones. Excerpt from zones.conf

```cat /etc/icinga2/zones.conf

object Endpoint "ossipemaster01a" {
  host = "10.xx.xx.173"
}

object Endpoint "ossipemaster01b" {
  host = "10.xx.xx.167"
}

object Zone "ap-on" {
  endpoints = ["ossipemaster01a","ossipemaster01b"]
}```


We have HA enabled successfully and the service checks are shared between 2 master nodes. However when we schedule downtimes, disable notifications for monitors we still get notifications.

Below is api.conf from primary master "a"

```
cat /etc/icinga2/features-available/api.conf
/**

The API listener is used for distributed monitoring setups.
*/
object ApiListener "api" { bind_host = "10.xx.xx.173" bind_port = 5665 ticket_salt = TicketSalt }
accept_config = false
accept_commands = false
```



api.conf from secondary master "b"

```
cat /etc/icinga2/features-available/api.conf
/**

The API listener is used for distributed monitoring setups.
*/
object ApiListener "api" { accept_config = true accept_commands = false }
```


message in log:


```
[2019-11-08 20:51:58 -0600] warning/ApiListener: Ignoring config update from 'ossipemaster01b.softlayer.local' for object 'compml1201d!compmlb
dal1201d-raidmon_c0_e8_s10_State!ossipemaster01b-1570119797-3' of type 'Comment'. 'api' does not accept config.

```

Looks like the scheduled downtimes, notification disabled are not sync'ed between master nodes causing lot of unwanted notifications


## Expected behavior

No notifications should be sent when downtimes, notifications are disabled



## Your Environment

Include as many relevant details about the environment you experienced the problem in

* Version used (`icinga2 --version`): 
root@ossipemastertok0201a:/var/log/icinga2# icinga2 --version
icinga2 - The Icinga 2 network monitoring daemon (version: r2.8.2-1)
* Operating System and version:Deb 9 stretch
* Enabled features (`icinga2 feature list`):
cinga2 feature list
Disabled features: debug debuglog elasticsearch gelf influxdb opentsdb perfdata statusdata syslog
Enabled features: api checker command compatlog graphite ido-mysql livestatus mainlog notification
* Icinga Web 2 version and modules (System - About):
* Config validation (`icinga2 daemon -C`):
No critical config errors
* If you run multiple Icinga 2 instances, the `zones.conf` file (or `icinga2 object list --type Endpoint` and `icinga2 object list --type Zone`) from all affected nodes.

## Additional context

Add any other context about the problem here.

**Comment by dnsmichi** [comment]
Hi,

first off, I'd suggest upgrading to 2.11.2 where a possible sync problem in this regard is fixed. Second to that, both masters need to set `accept_config = true` inside the api feature.

Cheers,
Michael

**Comment by hardoverflow** [comment]
We observe the same issue. Users ACK a service but notifications are still sent. On all cluster nodes, we have ```accept_config = true```.

**Comment by gangamaha** [comment]
@dnsmichi Not sure if setting accept_config=true on both master would result in split configs. According to icinga HA docs only secondary master should have accept_config=true right?

**Comment by dnsmichi** [comment]
Both masters need this setting being enabled. With 2.11, a loop bug has been fixed so it is safe to enable this flag again.

**Comment by gangamaha** [comment]
To clarify, our icinga version is 2.8. Are you suggesting to set accept_config=true for both masters in our version 2.8 or upgrade to icinga 2.11 first and then set accept_config=true on both master to fix the issue. Thanks!

**Comment by dnsmichi** [comment]
Upgrade to 2.11 first, 2.8 is not supported anymore. Then enable the flag and check the synchronisation of acknowledgements.

**Comment by gangamaha** [comment]
Closing this since the fix is in icinga 2.11 . we will upgrade and revert if issue revails. Thanks
